goal_src/engine/game/settings.gc

```diff
@@ -5,5 +5,734 @@
 ;; name in dgo: settings
 ;; dgos: GAME, ENGINE
 
-;; TODO - for video
-(define-extern *setting-control* setting-control)
+;; definition for method 9 of type setting-data
+(defmethod TODO-RENAME-9 setting-data ((obj setting-data) (arg0 engine))
+  (let ((s3-0 (-> arg0 alive-list-end))
```

Comment:
If you set `s3-0`'s type to `connection` in `var_names.json` it will make this much cleaner I think